server/internal/usecase/interfaces/item.go

```diff
@@ -4,20 +4,22 @@ import (
 	"context"
 	"errors"
 
+	"github.com/nicksnyder/go-i18n/v2/i18n"
 	"github.com/reearth/reearth-cms/server/internal/usecase"
 	"github.com/reearth/reearth-cms/server/pkg/id"
 	"github.com/reearth/reearth-cms/server/pkg/item"
 	"github.com/reearth/reearth-cms/server/pkg/key"
 	"github.com/reearth/reearth-cms/server/pkg/model"
 	"github.com/reearth/reearth-cms/server/pkg/schema"
 	"github.com/reearth/reearth-cms/server/pkg/value"
+	"github.com/reearth/reearthx/rerror"
 	"github.com/reearth/reearthx/usecasex"
 )
 
 var (
 	ErrItemFieldRequired   = errors.New("item field required")
 	ErrInvalidField        = errors.New("invalid field")
-	ErrDuplicatedItemValue = errors.New("duplicated value")
+	ErrDuplicatedItemValue = rerror.NewE(&i18n.Message{ID: "item:duplicated value", Other: "The inserted value is duplicated"})
```

Comment:
```suggestion
	ErrDuplicatedItemValue = rerror.NewE(&i18n.Message{ID: "duplicated item value", Other: "The inserted value is duplicated"})
```

It is better not to use `:` in a key

Reply:
I was thinking of a way to make the key refer to which model belongs. maybe that's more useful while translating the keys

Reply:
dot is better: `item.duplicated_value`